docs/specs/toc.yml

```diff
@@ -1106,21 +1106,25 @@ outputs:
   build.log: |
     ["error","output-path-conflict","Two or more files output to the same path 'docs/toc.json': 'docs/TOC.md', 'docs/TOC.yml'"]
 ---
-# validate bookmarks in toc	
-inputs:	
-  docfx.yml:	
-  docs/a.md: |	
-    # title 1	
-  docs/TOC.md: |	
-    # [link to title 1](a.md#title-1)	
-outputs:	
-  docs/a.json:	
-  docs/toc.json: |	
-    {	
-      "items": [	
-        { "name": "link to title 1", "href": "a#title-1", "!monikers":null },	
-      ]	
+# validate bookmarks in toc
```

Comment:
would you split this to another PR?

Reply:
Just added this test case back, the implementation was done in the previous PR